### PR TITLE
Outer Rim Galaxies patch

### PR DESCRIPTION
**Pull request by Person243546**
## Additions

Adds a compatibility patch for [O21] Outer Rim Galaxies [DevBuild].

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

**Comment by Breadbox2k19** [comment]
I would recommend the following tweaks for weapons.
-Long rifle could use a longer range, lower weapon spread, higher aim time and higher weapon sway.

File: Patches/[O21] Outer Rim Galaxies/Patch_OuterRim_Ammo.xml [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+						<defName>Bullet_OR_Blaster_Ion</defName>
+						<label>blaster bolt</label>
+						<graphicData>
+							<texPath>OuterRim/Projectile/BlasterBolt_Blue</texPath>
+							<graphicClass>Graphic_Single</graphicClass>
+						</graphicData>
+						<projectile Class="CombatExtended.ProjectilePropertiesCE">
+						  <damageDef>EMP</damageDef>
+						  <damageAmountBase>7</damageAmountBase>
+						  <!--<secondaryDamage>
```
Comment (MadaraUchiha) [inline Patches/[O21] Outer Rim Galaxies/Patch_OuterRim_Ammo.xml:340]: Please remove commented out code.

File: Patches/[O21] Outer Rim Galaxies/Clone Wars/Patch_OuterRim_CloneWars_Pawnkinds.xml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+								<max>12</max>
+							</primaryMagazineCount>
+							</li>
+						</value>
+					</li>
+					
+					</operations>
+				</match>
+			</li>
+			<!--
```
Comment (MadaraUchiha) [inline Patches/[O21] Outer Rim Galaxies/Clone Wars/Patch_OuterRim_CloneWars_Pawnkinds.xml:64]: Please remove commented out code.

**Review by MadaraUchiha — APPROVED** [review]
P-LGTM, please go over and remove commented out patches.

**Comment by N7Huntsman** [comment]
There are some conflicts to resolve.

**Comment by Person243546** [comment]
I couldn't figure out how to fix the conflict so I made a new pr

**Pull request closed without merging**